modules/datastore/modules/datastore_mysql_import/src/Service/MysqlImport.php

```diff
@@ -110,6 +118,31 @@ protected function runIt() {
     return $this->getResult();
   }
 
+  /**
+   * Attempt to detect the EOL character for the given line.
+   *
+   * @param string $line
+   *   Line being analyzed.
+   *
+   * @return string|null
+   *   The EOL character for the given line, or NULL on failure.
+   */
+  protected function getEol(string $line): ?string {
+    $eol = NULL;
```

Comment:
I think the default should be unix endings rather than null

```
TypeError: Argument 1 passed to Drupal\datastore_mysql_import\Service\MysqlImport::getEol() must be of the type string, null given, called in /var/www/dkan/modules/datastore/modules/datastore_mysql_import/src/Service/MysqlImport.php on line 101 in /var/www/dkan/modules/datastore/modules/datastore_mysql_import/src/Service/MysqlImport.php on line 130 #0 /var/www/dkan/modules/datastore/modules/datastore_mysql_import/src/Service/MysqlImport.php(101): Drupal\datastore_mysql_import\Service\MysqlImport->getEol(NULL)
```

Reply:
@janette I think the latest changes I just pushed should fix this error, but since I've thus far been unable to replicate it, I'll need you to confirm.